docs/source/types.rst

```diff
@@ -39,3 +103,109 @@ For ``Choice`` fields, their vocabulary or source will be linked to in a ``vocab
+
+To get one schema **fieldset** properties, access ``@types/Document/{fieldset}`` endpoint:
+
+..  http:example:: curl httpie python-requests
+    :request: ../../src/plone/restapi/tests/http-examples/types_document_get_fieldset.req
+
+.. literalinclude:: ../../src/plone/restapi/tests/http-examples/types_document_get_fieldset.resp
+   :language: http
+
+To get one schema **field** properties, access ``@types/Document/{field}`` endpoint:
+
+..  http:example:: curl httpie python-requests
+    :request: ../../src/plone/restapi/tests/http-examples/types_document_get_field.req
+
+.. literalinclude:: ../../src/plone/restapi/tests/http-examples/types_document_get_field.resp
+   :language: http
+
+
+Update schema with PATCH
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To update content type schema defaults we send a PATCH request to the server.
+PATCH allows to provide just a subset of the resource (the values you actually want to change).
+
+To update one or more schema **field** properties:
+
+..  http:example:: curl httpie python-requests
+    :request: ../../src/plone/restapi/tests/http-examples/types_document_patch_properites.req
+
+Response:
+
+.. literalinclude:: ../../src/plone/restapi/tests/http-examples/types_document_patch_properites.resp
+   :language: http
+
+To change one or more **fieldsets** properties:
+
+..  http:example:: curl httpie python-requests
+    :request: ../../src/plone/restapi/tests/http-examples/types_document_patch_fieldsets.req
+
+Response:
+
+.. literalinclude:: ../../src/plone/restapi/tests/http-examples/types_document_patch_fieldsets.resp
+   :language: http
+
+
+To update one **fieldset** settings, we can also send a PATCH request to ``@types/Document/{fieldset}`` endpoint:
+
+..  http:example:: curl httpie python-requests
+    :request: ../../src/plone/restapi/tests/http-examples/types_document_patch_fieldset.req
+
+Response:
+
+.. literalinclude:: ../../src/plone/restapi/tests/http-examples/types_document_patch_fieldset.resp
+   :language: http
+
+To update one **field** settings, we can also send a PATCH request to ``@types/Document/{field}`` endpoint:
+
+..  http:example:: curl httpie python-requests
+    :request: ../../src/plone/restapi/tests/http-examples/types_document_patch_field.req
+
+Response:
+
+.. literalinclude:: ../../src/plone/restapi/tests/http-examples/types_document_patch_field.resp
+   :language: http
+
+Update schema with PUT
+~~~~~~~~~~~~~~~~~~~~~~
+
+In accordance with the HTTP specification, a successful PUT will not create a new resource or produce a new URL.
+
+PUT expects the entire resource representation to be supplied to the server, rather than just changes to the resource state.
+This is usually not a problem since the consumer application requested the resource representation before a PUT anyways.
+
+When the PUT request is accepted and processed by the service, the consumer will receive a :term:`204 No Content` response (:term:`200 OK` would be a valid alternative).
```

Comment:
We should only mention the response we actually send here. So I guess this is 204.